test/unit/krane/kubernetes_resource_test.rb

```diff
@@ -177,11 +177,16 @@ def test_validate_definition_doesnt_log_raw_output_for_sensitive_resources
     resource = DummySensitiveResource.new
     kubectl.expects(:server_version).returns(Gem::Version.new('1.20'))
 
-    kubectl.expects(:run).with { |*_args, **kwargs| kwargs[:output_is_sensitive] == true }.returns([
-      "Some Raw Output",
-      "Error from kubectl: something went wrong and by the way here's your secret: S3CR3T",
-      stub(success?: false),
-    ])
+    kubectl.expects(:run)
```

Comment:
That's strange that this changed just from the Ruby version :/ I would have that it specific to Mocha. Regardless, I think it's ok for now, though I'm not a huge fan of introducing so much static info into the stub :(

Reply:
Ruby 2.7 deprecated some stuff with keyword args I haven't fully read up on it https://www.ruby-lang.org/en/news/2019/12/12/separation-of-positional-and-keyword-arguments-in-ruby-3-0/ but maybe this smells like it could be related.